File: v2/snippets/python/common/activation.py
from javonet.sdk import Javonet
from utils.activation_credentials import *


def test_activation_correct_credentials_0():
    # <Javonet_activate>
    result = Javonet.activate(ActivationCredentials.yourLicenseKey)
    # </Javonet_activate>
    assert (result == 0)

    # <Javonet_activate_without_credentials>
    result2 = Javonet.activate("")
    # </Javonet_activate_without_credentials>
    assert (result2 == 0)

File: v2/snippets/csharp/common/activation.cs
using Javonet.Netcore.Sdk.Tests.Utils;
using Xunit;
using Xunit.Abstractions;

namespace Javonet.Netcore.Sdk.Tests.common
{
    public class activation
    {
        private readonly ITestOutputHelper output;
        public activation(ITestOutputHelper output)
        {
            this.output = output;
        }

        [Fact]
        [Trait("Test", "Integration")]
        public void Test_Activation_CorrectCredentials_Returns0()
        {
            // <Javonet_activate>
            var result = Javonet.Activate(ActivationCredentials.yourEmail, ActivationCredentials.yourLicenseKey);
            // </Javonet_activate>
            Assert.Equal(0, result);

            // <Javonet_activate_without_credentials>
            var result2 = Javonet.Activate("", "");
            // </Javonet_activate_without_credentials>
            Assert.Equal(0, result2);
        }
    }
}